pkgs/development/tools/misc/jcli/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch }:
+
+buildGoModule rec {
+  pname = "jcli";
+  version = "0.0.41";
+
+  src = fetchFromGitHub {
+    owner = "jenkins-zh";
+    repo = "jenkins-cli";
+    rev = "v${version}";
+    hash = "sha256-oZtbjfquCYMMYgKe7sclRQDXnkNwAEY/GvwenGBnVk4=";
+  };
+
+  vendorHash = "sha256-bmPnxFvdKU5zuMsCDboSOxP5f7NnMRwS/gN0sW7eTRA=";
+
+  doCheck = false;
+
+  postInstall = ''
+    mv $out/bin/{jenkins-cli,jcli}
```

Comment:
```suggestion
    ln -s $out/bin/{jenkins-cli,jcli}
```

Reply:
It can conflict with official `jenkins-cli`:
https://github.com/NixOS/nixpkgs/blob/8501a1645abcc4ad91c9eadd53061124f5bebb3b/pkgs/development/tools/continuous-integration/jenkins/default.nix#L25-L26